source/docs/software/pathplanning/index.rst

```diff
@@ -0,0 +1,11 @@
+Path Planning
+=============
+
+Path Planning is the process of developing and "planning" paths that your robot can autonomously follow. These paths use the WPILib Trajectory Framework for generation and Ramsete for following. This section highlights the process of characterizing your robot for system identification, developing with the trajectory tracking API and visualizing your paths with PathWeaver. Users may also want to read the :ref:`generic trajectory following documents <docs/software/advanced-controls/trajectories/index:Trajectory Generation and Following with WPILib>` for non-commandbased usage.
```

Comment:
Did you mean to say a Ramsete Controller?  Just Ramsete doesn't really work here especially for someone who doesn't know what that is.